src/combinatorics/simple_graph/subgraph.lean

```diff
@@ -652,6 +812,25 @@ begin
     exact λ ha, ⟨G'.edge_vert ha, G'.edge_vert ha.symm⟩ }
 end
 
+lemma singleton_subgraph_eq_induce {v : V} :
+  G.singleton_subgraph v = (⊤ : G.subgraph).induce {v} :=
+by ext; simp [-set.bot_eq_empty, Prop.bot_eq_false] { contextual := tt }
+
+lemma subgraph_of_adj_eq_induce {v w : V} {hvw : G.adj v w} :
+  G.subgraph_of_adj hvw = (⊤ : G.subgraph).induce {v, w} :=
```

Comment:
I wonder if these make more sense in the opposite direction, potentially as simp lemmas? Also do you want `hvw` explicit here?

Reply:
I've made the argument explicit. I don't have much of a preference for which direction the lemma is stated -- I'll defer to you here.